subsys/mgmt/Kconfig.mcumgr

```diff
@@ -24,20 +24,74 @@ menuconfig MCUMGR_CMD_FS_MGMT
 	  Enables mcumgr handlers for file management
 
 if MCUMGR_CMD_FS_MGMT
+
+choice FS_MGMT_MAX_FILE_SIZE
+	prompt "Maximum file size that could be uploaded/downloaded"
+	default FS_MGMT_MAX_FILE_SIZE_64KB
+	default	FS_MGMT_MAX_FILE_SIZE_4GB
+	help
+	  Maximum file size that will be allowed to be downloaded from
+	  device.
+	  This option decides on number of bytes that are reserved in
+	  CBOR frame for storage of offset/size of file downloaded.
+
+config FS_MGMT_MAX_FILE_SIZE_64KB
+	bool "<= 64KB"
+	help
+	  Files that have size up to 64KB require 1 to 3 bytes to encode
+	  size/offset within CBOR frame with file chunk.
+
+config FS_MGMT_MAX_FILE_SIZE_4GB
+	bool "<= 4GB"
+	help
+	  Files that have size up to 4GB require 1 to 5 bytes to encode
+	  size/offset within CBOR frame with file chunk.
+
+endchoice
+
+config FS_MGMT_MAX_OFFSET_LEN
```

Comment:
This will be invisible option. Worth to comment that it has top be hidden.

Reply:
OK, will add proper comment.